website/src/components/canvas/options-bar/ShareCanvasModal.jsx

```diff
@@ -54,12 +55,14 @@ class ShareCanvasModal extends Component {
     this.state = {
       open: false,
       emailListText: null,
-      usersNotFound: null
+      usersNotFound: null,
+      snackbarOpen: false,
     };
     this.shareCanvas = this.shareCanvas.bind(this);
     this.updateEmailListText = this.updateEmailListText.bind(this);
     this.handleClose = this.handleClose.bind(this);
     this.handleOpen = this.handleOpen.bind(this);
+    this.handleSnackbarRequestClose = this.handleSnackbarRequestClose.bind(this);
```

Comment:
These 3 handlers taken together might be a bit confusing so if you could rename handleClose() and handleOpen() to something more descriptive that would be helpful!